Multisig/UI/Settings/OwnerKeyManagement/LedgerOwnerKey/LedgerKeyAddedViewController.swift

```diff
@@ -193,12 +193,29 @@ class AddDelegateKeyController {
     }
 
     func sendToBackend(delegateAddress: Address, signature: Data, completion: @escaping (Result<Void, Error>) -> Void) {
-        clientGatewayService.asyncCreateDelegate(safe: nil,
-                                                 owner: ownerAddress,
-                                                 delegate: delegateAddress,
-                                                 signature: signature,
-                                                 label: "iOS Device Delegate") { result in
-            completion(result.map { _ in () })
+        // to synchronize multiple async processes, we use DispatchGroup
+        let group = DispatchGroup()
+
+        Chain.all.forEach { chain in
+            // trigger request
+            group.enter()
+            clientGatewayService.asyncCreateDelegate(safe: nil,
+                    owner: ownerAddress,
+                    delegate: delegateAddress,
+                    signature: signature,
+                    label: "iOS Device Delegate",
+                    chainId: chain.id!) { result in
+                group.leave()
+            }
+        }
+        // wait until all requests finish, then call completion
+        let timeoutResult = group.wait(timeout: .now() + .seconds(60))
```

Comment:
should we move the magic number to a variable?

Reply:
Done in 3c6ddd62a6385fae46f4c467ed566ae18f6daf1e